anda/src/main.rs

```diff
@@ -1,54 +1,49 @@
-use clap::{arg, App, Arg, Command, SubCommand};
+use std::path::PathBuf;
+
+use clap::{AppSettings, Parser, Subcommand};
 use log::{debug, error, info, trace};
 use log4rs::*;
 
+#[derive(Parser)]
+#[clap(about, author, version)]
```

Comment:
Thanks for the PR! Can you help us with one more thing:
```suggestion
#[clap(about, version)]
```
The author list is pretty much obsolete, so it's good to remove it. By the way, Feel free to add yourself in `Cargo.toml`.

Reply:
Done!